web/src/containers/ApdStateKeyPerson.js

```diff
@@ -0,0 +1,250 @@
+import PropTypes from 'prop-types';
+import React, { Component, Fragment } from 'react';
+import { t } from '../i18n';
+import Dollars from '../components/Dollars';
+
+const tRoot = 'apd.stateProfile.keyPersonnel';
+
+const personTotalCost = person =>
+  Object.values(person.costs).reduce((sum, value) => sum + +value, 0);
+
+const PersonForm = ({
+  collapse,
+  number,
+  handleChange,
+  handleYearChange,
+  person,
+  primary,
+  years
+}) => {
+  const changeHasCosts = handleChange('hasCosts', true);
+
+  const toggleHasCosts = value => () => changeHasCosts({ target: { value } });
+
+  return (
+    <div className="ds-u-border-bottom--2 ds-u-margin-y--2 ds-u-padding-bottom--2">
+      <h3 className="ds-h3">
+        {primary ? 'Primary' : 'Additional'} APD Point of Contact
+      </h3>
+      <TextField
+        name={`apd-state-profile-pocname${number}`}
+        label={t(`${tRoot}.labels.name`)}
+        value={person.name}
+        onChange={handleChange('name')}
+      />
+      <TextField
+        name={`apd-state-profile-pocemail${number}`}
+        label={t(`${tRoot}.labels.email`)}
+        value={person.email}
+        onChange={handleChange('email')}
+      />
+      <TextField
+        name={`apd-state-profile-pocposition${number}`}
+        label={t(`${tRoot}.labels.position`)}
+        value={person.position}
+        onChange={handleChange('position')}
+      />
+      <TextField
+        name={`apd-state-profile-pocpercentTime${number}`}
+        label={t(`${tRoot}.labels.percentTime`)}
+        value={person.percentTime || 0}
+        size="small"
+        onChange={handleChange('percentTime')}
+      />
+
+      <FormLabel
+        label={t(`${tRoot}.labels.hasCosts`)}
+        id={`apd-state-profile-hascosts-label${number}`}
+      >
+        {t(`${tRoot}.labels.hasCosts`)}
+      </FormLabel>
+      <Choice
+        type="radio"
+        name={`apd-state-profile-hascosts-no${number}`}
+        value="no"
+        checked={!person.hasCosts}
+        onChange={toggleHasCosts(false)}
+      >
+        No
+      </Choice>
+      <Choice
+        type="radio"
+        name={`apd-state-profile-hascosts-yes${number}`}
+        value="yes"
+        checked={person.hasCosts}
+        onChange={toggleHasCosts(true)}
+        checkedChildren={
+          <div className="ds-c-choice__checkedChild ds-l-form-row">
+            {years.map(ffy => (
+              <div key={ffy} className="ds-l-col--auto">
+                <TextField
+                  name={`apd-state-profile-costs${number}-fy${ffy}`}
+                  label={`FFY ${ffy}`}
+                  size="small"
+                  mask="currency"
+                  value={person.costs[ffy]}
+                  onChange={handleYearChange(ffy)}
+                />
+              </div>
+            ))}
+            <div className="ds-l-col--auto">
+              <FormLabel>Total</FormLabel>
+              <div className="form-value-input-aligned">
```

Comment:
This is not an amazing class name, but I couldn't think of anything.  It's a computed form value that should be visually aligned like it's inside a text box but without being in a text box.